docs-devsite/vertexai.generationconfig.md

```diff
@@ -66,14 +66,24 @@ presencePenalty?: number;
 
 ## GenerationConfig.responseMimeType
 
-Output response mimetype of the generated candidate text. Supported mimetypes are `text/plain` (default, text output) and `application/json` (JSON response in the candidates). The model needs to be prompted to output the appropriate response type, otherwise the behavior is undefined. This is a preview feature.
+Output response MIME type of the generated candidate text. Supported MIME types are `text/plain` (default, text output) and `application/json` (JSON response in the candidates).
 
 <b>Signature:</b>
 
 ```typescript
 responseMimeType?: string;
 ```
 
+## GenerationConfig.responseSchema
+
+Output response schema of the generated candidate text. This value can be a class generated with a [Schema](./vertexai-preview.schema.md#schema_class) static method like `Schema.string()` or `Schema.object()` or it can be a plain JS object matching the [SchemaRequest](./vertexai-preview.schemarequest.md#schemarequest_interface) interface. <br/>Note: This only applies when the specified `responseMIMEType` supports a schema; currently this is limited to `application/json`<!-- -->.
```

Comment:
NOT BLOCKING

Schema and SchemaRequest

Both should be tagged as code in the source code. Pls look throughout the source code for other instances.


Anytime that you're referring to a specific component (method / field / enum / etc. name, file names, input / output value, etc.), then tag it as code.

One very generalized way to think about it to decide if something should be tagged as code: "Would I want this translated into German, Korean, etc.?" Note that our ref docs aren't translated, but this way of thinking helps you decide about appropriate tagging :-)

Reply:
I'm assuming by "tag as code" you mean to make it so it ends up in the markdown within `<code>` tags? This was pretty tricky, we normally do it with backticks but you can't do it on links so you normally choose between either having a link or having code tags. I think I found a way to do it although it's a little fragile. I did a search/replace on all `@link`s in the package.